client/lib/posts/actions.js

```diff
@@ -433,69 +434,77 @@ PostActions = {
 	*
 	* @api public
 	*/
-	fetchNextPage: function() {
-		var params, id, siteID;
+	fetchNextPage: function( postListStoreId = 'default' ) {
+		var params, id, siteID, postListStore;
 
-		if ( PostListStore.isLastPage() ) {
+		postListStore = postListStoreFactory( postListStoreId );
+
+		if ( postListStore.isLastPage() ) {
 			return;
 		}
 
 		Dispatcher.handleViewAction( {
-			type: 'FETCH_NEXT_POSTS_PAGE'
+			type: 'FETCH_NEXT_POSTS_PAGE',
+			postListStoreId: postListStoreId
 		} );
 
-		id = PostListStore.getID();
-		params = PostListStore.getNextPageParams();
-		siteID = PostListStore.getSiteID();
+		id = postListStore.getID();
+		params = postListStore.getNextPageParams();
+		siteID = postListStore.getSiteID();
 
 		if ( siteID ) {
 			wpcom
 				.site( siteID )
-				.postsList( params, PostActions.receivePage.bind( null, id ) );
+				.postsList( params, PostActions.receivePage.bind( null, id, postListStoreId ) );
 		} else {
 			wpcom
 				.me()
-				.postsList( params, PostActions.receivePage.bind( null, id ) );
+				.postsList( params, PostActions.receivePage.bind( null, id, postListStoreId ) );
 		}
 	},
 
-	receivePage: function( id, error, data ) {
+	receivePage: function( id, postListStoreId, error, data ) {
 		Dispatcher.handleServerAction( {
 			type: 'RECEIVE_POSTS_PAGE',
 			id: id,
+			postListStoreId: postListStoreId,
 			error: error,
 			data: data
 		} );
 	},
 
-	fetchUpdated: function() {
-		var id, params, siteID;
+	fetchUpdated: function( postListStoreId = 'default' ) {
+		var id, params, siteID, postListStore;
+
+		postListStore = postListStoreFactory( postListStoreId );
 
 		Dispatcher.handleViewAction( {
-			type: 'FETCH_UPDATED_POSTS'
```

Comment:
Looks like it is emitted [here](https://github.com/Automattic/wp-calypso/blob/master/client/lib/posts/actions.js#L471) and triggered by the poller in `post-list-fetcher` [here](https://github.com/Automattic/wp-calypso/blob/master/client/components/post-list-fetcher/index.jsx#L121).

Though honestly, we might consider an enhancement to the post-list-fetcher to disable polling in instances like the link selector in tinymce.